doc/api/cli.md

```diff
@@ -450,6 +450,14 @@ added: v0.8.0
 
 Silence deprecation warnings.
 
+### `--no-experimental-warnings`
+<!-- YAML
+added: v14.0.0
+-->
+
+Silence all experimental process warnings. These are emitted when using
+modules which are considered experimental.
```

Comment:
Instead of "modules" can you say "features" or "APIs" or something along those lines.